packages/ibm-products/src/components/Tearsheet/next/Tearsheet.stories.js

```diff
@@ -0,0 +1,564 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useRef, useState } from 'react';
+import {
+  AILabel,
+  AILabelContent,
+  Button,
+  Form,
+  FormGroup,
+  Heading,
+  IconButton,
+  NumberInput,
+  ProgressIndicator,
+  ProgressStep,
+  Section,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Tag,
+  TextInput,
+  Toggletip,
+} from '@carbon/react';
+
+import { Tearsheet } from '.';
+import styles from './_storybook-styles.scss?inline';
+import { Bee, RightPanelClose } from '@carbon/react/icons';
+import { TearsheetWithSteps } from './_story-assets/StepTearsheet';
+import { StepProvider } from '../../StepFlow';
+const storyClass = 'tearsheet-next-stories';
+
+export default {
+  title: 'Preview Candidate/Tearsheet',
```

Comment:
I'd suggest moving this to preview actually, since we've worked with design to completely rebuild it with composability and overall flexibility of the new api being a major focus.